docs/BR.md

```diff
@@ -1595,10 +1589,22 @@ The issuing CA SHALL document in its Certificate Policy or Certification Practic
 
 ## 7.3 OCSP profile
 
+1. All OCSP responses MUST conform with RFC6960 and be the profile specified in RFC5019.
+2. Responses MUST be pre-produced as specified in RFC5019.
+3. Responses MUST have the same value for the producedAt and thisUpdate times.
+4. The difference in time between the thisUpdate and nextUpdate fields MUST be greater than or equal to 8 hours and MUST NOT be greater than or equal to 192 hours (7 days).
+5. For OCSP responses directly signed by the issuing CA, the response MUST NOT include any certificates within the BasicOCSPResponse's certs field.
+6. For OCSP responses signed by a Delegated OCSP Responder Certificate, the response MUST only include the Delegated OCSP Responder Certificate within the BasicOCSPResponse's certs field and MUST NOT include any additional certificates.
+
 ### 7.3.1 Version number(s)
 
+OCSP responders MUST support v1 requests, as specified in RFC6960. OCSP responders MUST produce v1 responses, as specified in RFC6960.
+
 ### 7.3.2 OCSP extensions
 
+1. As specified in RFC5019 for pre-produced responses, responses MUST NOT include the id-pkix-ocsp-nonce extension.
+2. Responses MAY include an OCSP extension with the OID 1.3.6.1.4.1.11129.2.4.5. If such an extension is included, it MUST conform to RFC6962.
```

Comment:
This is just the CT extension, and making it a requirement that you can't put random stuff in here.